samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -903,6 +916,18 @@ static int pa_sync_past(struct bt_conn *conn, uint16_t pa_interval)
 	return err;
 }
 
+static void recv_state_updated_cb(struct bt_conn *conn,
+				  const struct bt_bap_scan_delegator_recv_state *recv_state)
+{
+	printk("Receive state updated, pa sync state: %d\n", recv_state->pa_sync_state);
+
+	for (int i = 0; i < recv_state->num_subgroups; i++) {
```

Comment:
```suggestion
	for (uint8_t i = 0; i < recv_state->num_subgroups; i++) {
```

I generally also suggest to use the same type for the loop variable as for the condition